{
    "PackageVulnerabilityID": "1",
    "DisclosureDate": "2021-02-13",
    "AffectedVersionRange": "*",
    "Summary": "Arbitrary Code Execution",
    "Contributor": {
        "Discloser": "61599526",
        "Fixer": ""
    },
    "Package": {
        "Registry": "other",
        "Name": "pystemon",
        "URL": "https://github.com/cvandeplas/pystemon",
        "Downloads": "0"
    },
    "CWEs": [
        {
            "ID": "94",
            "Description": "Arbitrary Code Execution"
        }
    ],
    "CVSS": {
        "Version": "3",
        "AV": "N",
        "AC": "L",
        "PR": "L",
        "UI": "N",
        "S": "U",
        "C": "H",
        "I": "H",
        "A": "H",
        "E": "X",
        "RL": "X",
        "RC": "X",
        "Score": "8.8"
    },
    "CVEs": [
        ""
    ],
    "Repository": {
        "URL": "https://github.com/cvandeplas/pystemon",
        "Codebase": [
            "Python"
        ],
        "Owner": "cvandeplas",
        "Name": "pystemon",
        "Forks": "229",
        "Stars": "283",
        "ForkName": "pystemon"
    },
    "Permalinks": [
        "https://drive.google.com/file/d/1z86y4CSHtckcxFQ0bgTsjUgYYLLGYWiX/view?usp=sharing"
    ],
    "References": [
        {
            "Description": "",
            "URL": ""
        }
    ],
    "PrNumber": "1895",
    "FixSubmissionCount": 1
}